src/components/MeetNotification/index.tsx

```diff
@@ -0,0 +1,187 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { io } from 'socket.io-client';
+import moment from 'moment';
+import { useAppDispatch, useAppSelector } from '@redux/hooks';
+import { socketAppointmentActions } from '@redux/slices/socketAppointmentsSlice';
+import cookie from 'utils/functions/cookies';
+import {
+  CallInfo,
+  DetailsBtn,
+  DoctorInfo,
+  El,
+  Info,
+  NotificationContainer,
+  PatientInfo,
+  Title,
+} from './styles';
+import { Appointment } from 'services/types/appointment.type';
+import { local } from '@constants/other';
+import { female } from '@constants/patient';
+import { notificationCurrentTime } from '@constants/format';
+import {
+  fiveMinutes,
+  MILLISEC_PER_SEC,
+  sec,
+  SEC_PER_MIN,
+} from '@constants/notification';
+
+const token = cookie.get('accessToken');
+
+type TimerResult = {
+  minutes: number;
+  seconds: number;
+};
+
+const MeetNotification = () => {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+  const [timer, setTimer] = useState(0);
+
+  const { patient, startTime, endTime, remoteDoctor, localDoctor } =
+    useAppSelector((state) => state.socketAppointmentReducer.nextAppointment);
+  const doctor = useAppSelector((state) => state.doctorReducer);
+
+  const formattedCurrentTime = useMemo(
```

Comment:
it look like could be a general function for full project not only for current component, its mean it should be moved to, for example, `utility` folder in the root folder